ruby/erm_buffer.rb

```diff
@@ -450,7 +457,16 @@ def on_op tok
 
     def on_period tok
       @mode ||= :period
+
       indent :c, tok.size if tok == "\n"
+
+      if @ident
+        line_so_far_str = @line_so_far.map {|(_, t, _)| t }.join
```

Comment:
`{ |a| a[1] }` might be clearer... but only maybe. Kinda wish for `map(&:second)`

But maybe rename `t` to `tok`?